src/wandbot/api/app.py

```
@@ -1,162 +1,295 @@
+        return {
+            "chat_config": safe_chat_config, 
+            "vector_store_config": safe_vs_config,
+            "git_info": git_info,
+            "app_config": app_config
+        }
+
+    except Exception as e:
+        raise HTTPException(
+            status_code=500,
+            detail=f"Error retrieving configs: {str(e)}"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)